controlplane/src/db/schema.ts

```diff
@@ -1621,11 +1621,7 @@ export const auditLogs = pgTable(
   'audit_logs', // auditlogs
   {
     id: uuid('id').notNull().primaryKey().defaultRandom(),
-    organizationId: uuid('organization_id')
-      .notNull()
-      .references(() => organizations.id, {
-        onDelete: 'cascade',
-      }),
+    organizationId: uuid('organization_id').notNull(), // we don't want the audit log to be dropped when the organization is deleted
```

Comment:
After deleting the org, how we will be able to identify an org in the logs?

Reply:
I have updated the audit logs table to include the `organization slug`,  that way we can at least identify the organization after being deleted. I opted for the slug as we already have it and is supposed to be unique.

Reply:
The migration also populates the field for the existing logs